### PR TITLE
0.8.0

### PR DESCRIPTION
**Pull request by piyush-kukadiya**
- core sdk v4.4.0 support
- push templates v1.0.0 support

### CODE REVIEW

File: Example/android/app/google-services.json [hunk 1]
```diff
@@ -1,33 +1,129 @@
 {
   "project_info": {
```
Comment (darshanclevertap) [inline Example/android/app/google-services.json:2]: @piyush-kukadiya please remove this file from the release

**Review by darshanclevertap — COMMENTED** [review]
Remove the `google-services.json` file. Rest LGTM 👍🏾

**Review by william-ct — APPROVED** [review]
LGTM👍

**Review by darshanclevertap — APPROVED** [review]
LGTM 👍🏾

**Comment by akashvercetti** [comment]
looks good

**Review by akashvercetti — APPROVED** [review]

**Pull request merged**